signin.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <title>로그인</title>
+    <meta
+      name="viewport"
+      content="width=device-width,minimum-scale=1,initial-scale=1,maximum-scale=1,user-scalable=no"
+    />
+    <link
+      rel="stylesheet"
+      type="text/css"
+      href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard/dist/web/static/pretendard.css"
+    />
+    <link rel="stylesheet" href="./src/sign.css" />
+  </head>
+  <body>
+    <header>
+      <a class="logo-link" href="index.html">
+        <img class="header-logo" src="./images/logo.svg" alt="홈으로 연결된 Linkbrary 로고" />
+      </a>
+      <p class="header-message">
+        회원이 아니신가요?<a class="header-link" href="signup.html">회원 가입하기</a>
+      </p>
+    </header>
+    <div class="sign-box">
+      <form class="sign-form" method="post" action="#" onsubmit="checkAccount()">
```

Comment:
form 요소에 onsubmit 핸들러를 직접 등록하셨군요!  
onsubmit 핸들러를 통해 폼의 동작을 직접 제어하고 계시니, 아래와 같이 `checkAccount` 함수를 호출할 때 event 객체를 전달해주어야 합니다.  
```suggestion
      <form class="sign-form" method="post" action="#" onsubmit="checkAccount(event)">
```